cfp/models.py

```diff
@@ -1,8 +1,13 @@
 from django.db import models
 
+
 class Proposal(models.Model):
     first_name = models.CharField(verbose_name="First Name", max_length=50)
     last_name = models.CharField(verbose_name="Last Name", max_length=50)
+    email = models.EmailField(verbose_name="Email Adress", max_length=50)
+    profile_link = models.URLField(verbose_name="GitHub/Home Page", max_length=50)
+    phone = models.CharField(verbose_name="Phone Number", max_length=12)
```

Comment:
I wouldn't cap to 12, consider speakers from outside the US that want to also add country code. Do you think it's worth making it bigger?